Raw json:

{'hasError': True, 'errParameter': ['dfs.datanode.volumes.replica-add.threadpool.size', 'datanode.https.port'], 'reason': ["The value for 'dfs.datanode.volumes.replica-add.threadpool.size' should be an integer representing the maximum number of threads. A value of 0.5 is invalid because thread count cannot be a fraction.", "The value for 'datanode.https.port' is 'ciri', which is invalid. It should be an integer representing the port number."]}

Final result:

There are 2 misconfiguration parameters in the input: dfs.datanode.volumes.replica-add.threadpool.size	datanode.https.port
[Ciri] Reason for dfs.datanode.volumes.replica-add.threadpool.size: The value for 'dfs.datanode.volumes.replica-add.threadpool.size' should be an integer representing the maximum number of threads. A value of 0.5 is invalid because thread count cannot be a fraction.
[Ciri] Reason for datanode.https.port: The value for 'datanode.https.port' is 'ciri', which is invalid. It should be an integer representing the port number.
